src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/CollectionDeserializationHelpers.cs

```diff
@@ -119,10 +119,35 @@ public static void Clear(this IEnumerable collection, string propertyName, Type
             clearMethod.Invoke(collection, _emptyObjectArray);
         }
 
-        public static bool TryCreateInstance(Type collectionType, IEdmCollectionTypeReference edmCollectionType, Type elementType, out IEnumerable instance)
+        /// <summary>
+        /// Creates an instance of a collection object. 
+        /// </summary>
+        /// <param name="collectionType">The <see cref="Type"/> of the collection.</param>
```

Comment:
nit: I don't know I feel about `<see cref="Type"/>` across your PR. Methinks that you should use the word "type" instead. I cant imagine anyone navigating to the definition of "Type" to learn what that is or what it represents - which is basically what a `cref` helps someone do. It's however not a deal breaker. It however feels/looks odd/superfluous